tests/integration/cases/github_maven_attestation_local/test.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+description: |
+  Discovering GitHub attestation of a local Maven artifact but failing because the artifact is wrong
+
+tags:
+- macaron-python-package
```

Comment:
I wonder if we could also run this test as part of the Docker integration test. There is extra handling of the local artifact directories for the Docker container so we want to make sure it's working fine too.